spec/lib/saml/url_service_spec.rb

```diff
@@ -5,20 +5,20 @@
 require 'support/url_service_helpers'
 
 RSpec.describe SAML::URLService do
-  SAML::URLService::VIRTUAL_HOST_MAPPINGS.each do |vhost_url, values|
-    subject { described_class.new(saml_settings, session: session, user: user, params: params) }
+  subject { described_class.new(saml_settings, session: session, user: user, params: params) }
 
-    let(:user) { build(:user) }
-    let(:session) { Session.create(uuid: user.uuid, token: 'abracadabra') }
+  let(:user) { build(:user) }
+  let(:session) { Session.create(uuid: user.uuid, token: 'abracadabra') }
 
-    around do |example|
-      User.create(user)
-      Timecop.freeze('2018-04-09T17:52:03Z')
-      RequestStore.store['request_id'] = '123'
-      example.run
-      Timecop.return
-    end
+  around do |example|
+    User.create(user)
+    Timecop.freeze('2018-04-09T17:52:03Z')
+    RequestStore.store['request_id'] = '123'
+    example.run
+    Timecop.return
+  end
 
+  SAML::URLService::VIRTUAL_HOST_MAPPINGS.each do |vhost_url, values|
```

Comment:
Git is showing this diff weirdly... all I did was move the `let` and `around` declarations outside of the `SAML::URLService::VIRTUAL_HOST_MAPPINGS.each`, so that I could re-use them at the bottom of the file.